components/Proposals/ProposalDetailCard.tsx

```diff
@@ -57,15 +58,11 @@ export function ProposalDetailCard() {
   const [isRedeeming, setIsRedeeming] = useState<boolean>(false);
 
   const remainingSlots = useMemo(() => {
-    if (!proposal || !markets || !daoState) return;
```

Comment:
this is what was causing the issue - we were returning null when we didn't have `markets` but they weren't in the dependencies of the memo